### DISCUSSION TITLE
Unreleased images?

### DISCUSSION THREAD

**Discussion by artem-nefedov** [argoproj/argo-rollouts#1452] [Q&A] [answered]
Our automated dependency update bot detected that docker images with tags v1.0.5 and v1.0.6 now exists on quay repo. However, there are no such releases on github, and files https://github.com/argoproj/argo-rollouts/releases/download/v1.0.5/install.yaml and https://github.com/argoproj/argo-rollouts/releases/download/v1.0.6/install.yaml do not exist. Is this intentional? I'm asking to decide if we should switch the source of updates from docker images to github releases.

**Comment by jessesuen** [comment, marked as answer]
I think you were looking at the pages at the inopportune time. The sequence of releasing is:
1. git SHA is tagged
2. container image and manifests are built from the tag
3. when completed, release is published to github

I think you happened to catch us between steps 2 and 3. As of now, v1.0.6 is "officially" [released](https://github.com/argoproj/argo-rollouts/releases/tag/v1.0.6).

What happened with v1.0.5, is that it introduced a major bug that warranted us to pull the release down (replicas were flapping). Because of that, we did not want to encourage any more installations of v1.0.5 and quickly pushed out v1.0.6

> I'm asking to decide if we should switch the source of updates from docker images to github releases.

Given our current release process, if I were you, I **_would_** switch to github releases as the source of truth for updates over quay.
